config/parser.go

```diff
@@ -0,0 +1,74 @@
+package config
+
+import (
+    "io/ioutil"
+    "encoding/json"
+
+    "./../error"
+)
+
+const path = "./config.json"
```

Comment:
Config path should be as command line argument and not fixed constant. 
`configPath := flag.String("config", "config.json", "Default config file")`